test/depositAndExitTxs.js

```diff
@@ -181,7 +189,7 @@ contract('Deposit and Exit Transactions', async (accounts) => {
         // finalize
         let balance, contractBalance, childChainBalance;
         [balance, contractBalance, childChainBalance]
-            = await rootchainHelpers.successfulFinalizeExit(rootchain, accounts[2], authority, blockNum, depositAmount, minExitBond, true);
+            = await rootchainHelpers.successfulFinalizeExit(rootchain, accounts[2], authority, blockNum, depositAmount, minExitBond, 1, true);
```

Comment:
Can we have a test where there are 2 exits in the queue, you pass in numIter = 1 and test that only the first exit got processed?